server/test/ping.spec.js

```diff
@@ -1,21 +1,20 @@
 const chai = require("chai");
 const chaiHttp = require("chai-http");
-const app = require("../app.js");
+const {app} = require("../app");
+let should = chai.should();
```

Comment:
```suggestion
const should = chai.should();
```
I don't see the reassignment of this variable :)